tasks/patternpack.js

```diff
@@ -134,6 +139,24 @@ module.exports = function (grunt) {
     }
   }
 
+  function ensureFilesExist(options) {
+    var path = require("path");
+
+    // Create core css file if it does not exist
+    var coreCssPath = options.assets + "/" + options.css.preprocessor + "/" + options.css.fileName + "." + options.css.preprocessor;
+    if (!fs.existsSync(coreCssPath)) {
+      fs.writeFileSync(coreCssPath, '@import "_patternpack-patterns"');
+    }
+
+    // Create the dirctories for the pattern structure if they do not exist
+    _.each(options.patternStructure, function(pattern) {
+      var patternPath = options.src + "/" + pattern.path;
+      if (!fs.existsSync(patternPath)) {
+        fs.mkdir(patternPath);
+      }
+    });
```

Comment:
Thoughts on also generating a `src/index.md` here? It could be a placeholder for 1.0.2, but eventually I thought this could be a great place to put a good "first run" experience with a "Welcome to PatternPack" page that outlines the next steps.

For the sake of getting 1.0.2 out, it can be a blank file:

```
---
title: Welcome to PatternPack
---

# Welcome to PatternPack!
```